src/video/SDL_blit_A.c

```diff
@@ -1285,6 +1179,15 @@ static void BlitNtoNSurfaceAlphaKey(SDL_BlitInfo *info)
     }
 }
 
+/* Accurate alpha blending with no division */
+static Uint8 AlphaBlendChannel(Uint8 sC, Uint8 dC, Uint8 sA)
+{
+    Uint16 x = ((sC - dC) * sA) + ((dC << 8) - dC);
+    x += 0x1U; // Use 0x80 to round instead of floor
+    x += x >> 8;
+    return x >> 8;
+}
```

Comment:
Is this faster as a macro?

Reply:
Haven't measured, but it makes more sense as a macro with recent changes I've mentioned elsewhere. Updated.